packages/client/jest.config.ts

```diff
@@ -0,0 +1,19 @@
+import type { Config } from '@jest/types'
+import { defaults as tsjPreset } from 'ts-jest/presets'
+
+const config: Config.InitialOptions = {
+  preset: 'ts-jest',
+  projects: [
+    {
```

Comment:
I added tests for the client so we can tests our routes. It's a sort of semi-automatic testing like the socket tests. You need to start a server on port 3100 before running the tests